pkgs/development/compilers/openspin/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "openspin-unstable-${version}";
```

Comment:
Since you're calling it "openspin" in top level, I would prefer to lose to "unstable" part her

Reply:
Actually, this is part of the package naming policy: https://nixos.org/nixpkgs/manual/#sec-package-naming